config/betterquesting/DefaultQuests.json

```diff
@@ -6608,7 +6618,7 @@
       "properties:10": {
         "betterquesting:10": {
           "autoclaim:1": 0,
-          "desc:8": "Convert 1 log into 4 §6planks§r. \n\nYou can then craft those planks into a §acrafting table§f.\n\n----------------------------------------\n§3WELCOME TO SUPERSYMMETRY!\n§r----------------------------------------\n- Explore your surroundings\n§r- Find or build shelter\n- Prepare for the winter\n- Acquire basic resources\n- Try not to die\n------------------------------------",
+          "desc:8": "Make a §6Crafting Table§f\n\n----------------------------------------\n§3WELCOME TO SUPERSYMMETRY!\n§r----------------------------------------\n- Explore your surroundings\n§r- Find or build shelter\n- Prepare for the winter\n- Acquire basic resources\n- Try not to die\n------------------------------------",
```

Comment:
why

Reply:
I think if you are playing susy you know how to make a crafting table but my text could be changed for smth more poetic